PyDSS/dssInstance.py

```diff
@@ -346,7 +379,8 @@ def RunSimulation(self, project, scenario, MC_scenario_number=None):
         Steps, sTime, eTime = self._dssSolver.SimulationSteps()
         self._Logger.info('Running simulation from {} till {}.'.format(sTime, eTime))
         self._Logger.info('Simulation time step {}.'.format(Steps))
-        if self._Options['Exports']['Result Container'] == 'ResultData' and self.ResultContainer is not None:
+        if self._Options['Exports']['Result Container'] == 'ResultData':
+            print("initializing store")
```

Comment:
I doubt we need this.  If we do, it should be a log message.

Reply:
I need this implementation for NAERM, where a local store is not used.

Reply:
then make it a log message?

Reply:
done